core/lib/src/fs/mod.rs

```diff
@@ -11,6 +11,7 @@ pub use server::*;
 pub use named_file::*;
 pub use temp_file::*;
 pub use file_name::*;
+pub use rewrite::{Rewrite, Rewriter};
```

Comment:
I don't think we should export the same type from two different spaces. My view is this: if one is going to use `Rewrite` or `Rewriter`, then they'll likely also use one of the existing rewriters. As such, it makes sense to either expose all `rewrite` in `fs` or _none_ of `rewrite` in `fs`. For clarity, I would prefer the latter.